src/components/organisms/AdminVenueView/AdminVenueView.tsx

```diff
@@ -103,8 +105,12 @@ export const AdminVenueView: React.FC = () => {
     [history, venueId]
   );
 
-  if (!isCurrentVenueLoaded) {
-    return <LoadingPage />;
+  if (!venueId) {
+    return <NotFound />;
+  }
+
+  if (!venue) {
+    return isCurrentVenueLoaded ? <NotFound /> : <LoadingPage />;
```

Comment:
`<NotFound fullscreen/>` ?

Reply:
That will cover the NavBar which is still useful for navigating to beginning or clicking the profile to log out.